Refresher/Patching/PSPPatcher.cs

```diff
@@ -58,10 +58,10 @@ public void Patch(string url, bool patchDigest)
 
         //Match for all files called "game.txt" in the plugins directory
         //NOTE: we do this because the PSP filesystem is case insensitive, and the .NET STL is case sensitive on linux
-        List<string> possibleMatches = Directory.EnumerateFiles(pluginsDir, "game.txt", new EnumerationOptions
+        IEnumerable<string> possibleMatches = Directory.EnumerateFiles(pluginsDir, "game.txt", new EnumerationOptions
```

Comment:
Why change to `IEnumerable` here? This could cause multiple enumerations since we call `.First()` twice later. Should be a `Collection` or something at minimum.

Could also avoid LINQ entirely in this function.

Reply:
https://github.com/LittleBigRefresh/Refresher/pull/31/commits/908cfb114f852b660b8f06bf9dca354b3d3cd602